sounds/management/commands/csv_bulk_upload.py

```diff
@@ -119,6 +120,9 @@ def handle(self, *args, **options):
                 sound.original_path = new_original_path
                 sound.save()
 
+            # Copy to mirror location
+            copy_files_to_mirror_locations(sound, ['SOUNDS'])
```

Comment:
The usage of magic string variables is kind of ugly. You could misspell them and python will not tell you anything. You could write "SOUDNS".

Perhaps this could be an idea:

*settings.py*

```
MIRROR_SOUNDS = "SOUNDS"
MIRROR_PREVIEWS = "PREVIEWS"
MIRROR_DISPLAYS = "DISPLAYS"
MIRROR_ANALYSIS = "ANALYSIS"

MIRROR_DISK_LOCATIONS = {
    MIRROR_SOUNDS: None,
    MIRROR_PREVIEWS: None,
    MIRROR_DISPLAYS: None,
    MIRROR_ANALYSIS: None
 }
```

*csv_bulk_upload.py*

```
copy_files_to_mirror_locations(sound, [settings.MIRROR_SOUNDS])
```

Event nicer would be using enums: https://docs.python.org/3/library/enum.html
They were back-ported to python 2.X: https://pypi.python.org/pypi/enum34